### PR TITLE
[Blog] Week 24 (June 8th - June 11th)

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

File: content/posts/June-2020.md [hunk 1]
```diff
@@ -10,10 +10,23 @@ Hello everyone, during this week we didn't manage to implement any significant n
...
+
+## Week 24 (June 8th - June 11th)
+
+### packit
+
+* [Tomas](https://github.com/TomasTomecek) re-implemented overriding of package config per job, which would also enable you to set up different configurations for builds, releases, etc. ([#858](https://github.com/packit-service/packit/pull/858), [#859](https://github.com/packit-service/packit/pull/859)).
+
+### packit-as-a-service
+
+* We are now parsing events from Koji about builds` progress and report them (by [Franta](https://github.com/lachmanfrantisek/), [#657](https://github.com/packit-service/packit-service/pull/657)).
```
Comment (csomh) [inline content/posts/June-2020.md:30]:
```suggestion
* We are now parsing events from Koji about the progress of the builds and report them (by [Franta](https://github.com/lachmanfrantisek/), [#657](https://github.com/packit-service/packit-service/pull/657)).
```

File: content/posts/June-2020.md [hunk 2]
```diff
@@ -10,10 +10,23 @@ Hello everyone, during this week we didn't manage to implement any significant n
...
 ### packit-as-a-service
 
 * We have run into multiple problems when you set up jobs for PRs and branch pushes - there were multiple fixes for this, packit-service wasn't able to work well with the fact that there are multiple jobs defined for the same job type - e.g. build.
+
+
+## Week 24 (June 8th - June 11th)
+
+### packit
+
+* [Tomas](https://github.com/TomasTomecek) re-implemented overriding of package config per job, which would also enable you to set up different configurations for builds, releases, etc. ([#858](https://github.com/packit-service/packit/pull/858), [#859](https://github.com/packit-service/packit/pull/859)).
```
Comment (csomh) [inline content/posts/June-2020.md:26]: If I understand it correctly, this is not ready to be used yet, as it requires packit-service/packit-service#676. So maybe rephrase this as "... config per job. This will enable overriding configuration for builds and releases, for example, once the corresponding changes are merged in Packit-as-a-Service."

File: content/posts/June-2020.md (hunk 1 above)
Comment (jpopelka) [reply content/posts/June-2020.md]: amended

File: content/posts/June-2020.md (hunk 2 above)
Comment (jpopelka) [reply content/posts/June-2020.md]: amended
